[DOCS-6134] AGS Community

oliveralfresco2018 opened the pull request.

anxumalo reviewed (changes requested):
Looks good but I have some questions:

- There are links to `governance-services/latest/` - should they all be `governance-services/community/`?
- `governance-services/community/install/docker.md` - line 18: this links to an old archived repo (ACS Community Deployment). The Docker Compose file for Community deployments has been renamed. See new file in https://github.com/Alfresco/acs-deployment/blob/master/docker-compose/community-docker-compose.yml.
- `governance-services/community/develop/index.md` - line 15: has the Alfresco Community RM ReST API Explorer version changed with release? It still says `alfresco-rm-community-rest-api-explorer-3.0.a.war`.

oliveralfresco2018 commented:
I was required to add the SMC page which was missing from the migration.
The links that were 'latest' were pointing there because the page didnt exist in the comm version for whatever reason. It is an old page so I've just added it and updated all the links to be comm.
I think this is good to go know.

oliveralfresco2018 commented on governance-services/community/develop/index.md:
```diff
@@ -10,7 +10,7 @@ The APIs are designed for you to create remote clients to manage the {% include
 You can download the API Explorer from:
 
-* [Alfresco Community RM ReST API Explorer](https://download.alfresco.com/cloudfront/release/community/RM/3.0.a/alfresco-rm-community-rest-api-explorer-3.0.a.war)
+* [Alfresco Community RM ReST API Explorer](https://download.alfresco.com/cloudfront/release/community/RM/3.0.a/alfresco-rm-community-rest-api-explorer-3.5.a.war)
```
Updated to https://download.alfresco.com/cloudfront/release/community/RM/3.5.a/alfresco-governance-services-community-rest-api-explorer-3.5.a.war
